tests/test_middleware_interface.py

```diff
@@ -567,3 +580,27 @@ def test_export_outputs_bad_visit(self):
     def test_export_outputs_bad_exposure(self):
         with self.assertRaises(ValueError):
             self.interface.export_outputs(self.next_visit, {88})
+
+    def test_export_outputs_retry(self):
+        self.interface.export_outputs(self.next_visit, {self.raw_data_id["exposure"]})
+
+        time.sleep(1.0)  # Force _simulate_run() to create a new timestamped run
+        self._simulate_run()
+        self.interface.export_outputs(self.second_visit, {self.raw_data_id["exposure"]})
+
+        central_butler = Butler(self.central_repo.name, writeable=False)
+        raw_collection = f"{instname}/raw/all"
+        export_collection = f"{instname}/prompt-results"
+        self._check_datasets(central_butler,
+                             "raw", raw_collection, 2, self.raw_data_id)
+        # Did not export raws directly to raw/all.
+        self.assertNotEqual(central_butler.registry.getCollectionType(raw_collection), CollectionType.RUN)
```

Comment:
Do we expect this collection to be a chained collection? Why not `assertEqual` that instead?